repositories/analytics_views/apikeys.sql

```diff
@@ -0,0 +1,10 @@
+DROP VIEW IF EXISTS analytics.apikeys;
+
+CREATE VIEW analytics.apikeys WITH (security_invoker = false) AS
```

Comment:
I do "drop" then "create", because "CREATE OR REPLACE" does not allow for schema updates (e.g. you can't rename a column on an existing view doing so)